spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    author = User.create(name: 'author', photo: 'photo', bio: 'bio', posts_counter: 2)
+    post = Post.create(title: 'title', text: 'text', comments_counter: 2, author_id: author.id, likes_counter: 2)
+    subject { Like.new(author_id: 1, post_id: 1) }
+
+    before { subject.save }
+
+    it 'author id should be an integer' do
+      subject.author_id = 'string'
+      expect(subject).to_not be_valid
+    end
+
+    it 'post id should be an integer' do
+      subject.post_id = 'string'
+      expect(subject).to_not be_valid
+    end
+
+    it 'updates the likes counter' do
+      expect(post.likes_counter).to eq(2)
+    end
+  end
```

Comment:
- [ ] Hello there, @bernabasy. You made an excellent effort to correct the failing tests as requested by the previous reviewer :+1: . However, I continue to receive the same error. :point_down: 
![Screenshot from 2023-04-07 16-10-14](https://user-images.githubusercontent.com/89228743/230641756-d9a3af8f-12be-410b-a46d-9226ef09b88e.png)

- [ ] After further investigation, I discovered that the error is coming from the `Like` model. The error occurs because you are passing the `likes counter` as if it were a variable that hasn't been declared anywhere. As a result, we get the error `undefined local variable or method `likes counter`. The `likes counter`, on the other hand, should be passed as a `string` or a `symbol`. So take a look at my suggestion for the `Like` model.

- [ ] We can fix the error like this :point_down: 
` post.increment!(:likes_counter)` or this :point_down: 
` post.increment!("likes_counter")`